### DISCUSSION TITLE
Flex Dog

### DISCUSSION THREAD

**Discussion by jasonmhead** [jasonmhead/open-tentacle#20] [Ideas]
Flex dog is an application concept for use of Open Tentacle as legs.
Concept renders can be found here until the repo is updated: https://headrobotics.co/flex-dog/

**Comment by cnuland** [comment]
Will the STL or OBJ file be available for this one? I thought this might be a good candidate to test out some casting equipment I'm getting and do a resin mould of the base of this robot.

**Reply by jasonmhead** [reply]
I've reinitialized / redone the repository that now includes a body STL and OBJ file in https://github.com/jasonmhead/open-tentacle/tree/main/concepts/flex_dog

There are some geometry flaws that will/could cause some issues depending on how you use it 
(for example if you try a 3D print it will need manifold cleaned first - could be done with something like these: https://all3dp.com/1/stl-repair-stl-file-online-checker-fix-3d-model/ --  this one could be pretty good: https://makeprintable.com/)
